test/lib/themingParameters/dataUri.js

```diff
@@ -0,0 +1,159 @@
+/* eslint-env mocha */
+
+const assert = require("assert");
+// const sinon = require("sinon");
+
+// tested module
+const themingParametersDataUri = require("../../../lib/themingParameters/dataUri");
+
+describe("themingParameters/dataUri", function() {
+	it("should not add theming parameters when library name is missing", function() {
+		const result = {};
```

Comment:
this initializer made me aware of the fact that `dataUri` will fail if the `options` indicate to act, but one of `result.{css,rtl,cssVariables}` is null or undefined.

Reply:
Correct. It doesn't fail, but produces a string starting with `undefined` (or `null`).
The contract was that if options are enabled, the results properties must be set. Right now I would stick with that. Handling this case and throwing an error would be a good improvement.